### PR TITLE
Implemented individual ad page commment section using disqus

### PR DESCRIPTION
**Pull request by BrandonBalala**
#183

### CODE REVIEW

**Comment by kevcamel** [comment]
screen shot 2017-11-02 at 12 31 13 am

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -129,5 +129,6 @@
 <script src="{{ asset('js/jquery/jquery.min.js') }}"></script>
 <script src="{{ asset('js/tether/tether.min.js') }}"></script>
 <script src="{{ asset('js/bootstrap/bootstrap.min.js') }}"></script>
+<script id="dsq-count-scr" src="//classifiedz-xyz.disqus.com/count.js" async></script>
```
Comment (kevcamel) [inline resources/views/layouts/app.blade.php:132]: what does the async do?

File: resources/views/viewAd.blade.php [hunk 2]
```diff
@@ -20,6 +20,29 @@
...
+        };
+        
+        (function() { // DON'T EDIT BELOW THIS LINE
+        var d = document, s = d.createElement('script');
+        s.src = 'https://classifiedz-xyz.disqus.com/embed.js';
+        s.setAttribute('data-timestamp', +new Date());
+        (d.head || d.body).appendChild(s);
+        })();
+        </script>
+        <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
```
Comment (kevcamel) [inline resources/views/viewAd.blade.php]: I had no idea this no script tag existed

**Review by kevcamel — COMMENTED** [review]
Good implementation using Disqus 👍 Lines 33-34 and 38-41 should be indented.

**Comment by kevcamel** [comment]
@BrandonBalala once you indent those lines we could merge this PR

File: resources/views/viewAd.blade.php (hunk 2 above)
Comment (neqqash) [reply resources/views/viewAd.blade.php]: neither did i, i guess we learn something new everyday

File: resources/views/layouts/app.blade.php (hunk 1 above)
Comment (BrandonBalala) [reply resources/views/layouts/app.blade.php:132]: I have no clue, but this line of code was given by disqus.

**Comment by neqqash** [comment]
Disqus FTW

**Pull request merged**